app/components/coding-exercise/lib/orchestrator/EditorFacade.ts

```diff
@@ -0,0 +1,81 @@
+import type { EditorView } from "@codemirror/view";
+import type { StoreApi } from "zustand/vanilla";
+import type { EditorManager } from "./EditorManager";
+import type { OrchestratorStore } from "../types";
```

Comment:
I don't see why we need this at all. We have an EditorManager, that's already designed for this.